rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -171,7 +172,7 @@ def ensure_global(name, default):
 #
 ## rosdoc2 will override the theme, but you may set one here for running Sphinx
 ## without the rosdoc2 tool.
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
So if the user copies this and then runs without rosdoc2, this will fail I think, since it isn't in the extensions?

Reply:
Or does it just need to be installed? in which case adding it to the extensions above isn't right?

Reply:
No, we need to add it to the extensions.  Good catch.